db/sql/buildings_reference/functions/08-suburb_locality.sql

```diff
@@ -77,15 +85,71 @@ $$
 LANGUAGE sql VOLATILE;
 
 COMMENT ON FUNCTION buildings_reference.bulk_load_outlines_update_suburb(integer) IS
-'Replace suburb values with the intersection result';
+'Replace suburb values with the intersection result of buildings from a supplied dataset in the bulk_load_outlines table';
+
+-- bulk_load_outlines_update_all_suburbs (replace all suburb values with the intersection result)
+    -- params: integer[] list of suburb localities building must be within
+    -- return: count(integer) number of building outlines updated
+CREATE OR REPLACE FUNCTION buildings_reference.bulk_load_outlines_update_all_suburbs(integer[])
```

Comment:
I think it should be a function in buildings_bulk_load. Same for the other bulk_load_outlines&building outlines